src/content/configuration/configuration-languages.mdx

```diff
@@ -22,8 +23,6 @@ To write the webpack configuration in [TypeScript](http://www.typescriptlang.org
 
 ```bash
 npm install --save-dev typescript ts-node @types/node @types/webpack
-# and, if using webpack-dev-server
-npm install --save-dev @types/webpack-dev-server
```

Comment:
```diff
- # and, if using webpack-dev-server
+ # and, if using webpack-dev-server < v4.7.0
```
Let's just update the comment for now. Since, it is released recently.